src/features/defi/providers/univ2/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+
+export function calculateSlippageMargin(amount: string | null, precision: number) {
+  if (!amount) throw new Error('Amount not given for slippage')
+  const percentage = '0.5'
```

Comment:
really want to hear from @willyogo what we want to do here product wise - was previously hardcoded to 3% for adding and removing liquidity

Reply:
0.5% LGTM 🚀 

That's what Uniswap v2 has always defaulted to
<img width="414" alt="image" src="https://user-images.githubusercontent.com/26232238/231582811-7266f15c-0692-4972-9b8c-9a72c0736da8.png">